libs/application-generic/src/usecases/get-preferences/get-preferences.usecase.ts

```diff
@@ -94,11 +98,7 @@ export class GetPreferences {
   /** Transform WorkflowChannelPreferences into IPreferenceChannels */
   public static mapWorkflowChannelPreferencesToChannelPreferences(
     workflowPreferences: WorkflowChannelPreferences,
-  ): IPreferenceChannels | undefined {
-    if (!workflowPreferences) {
-      return undefined;
-    }
-
+  ): IPreferenceChannels {
```

Comment:
The changes in this file ensure the `mapWorkflowChannelPreferencesToChannelPreferences` will only be used for defined `workflowPreferences`